articles/Frontend/Architecture/coding-guide.md

```diff
@@ -263,19 +258,19 @@ For example,
 ```js
 //the mixin, a separate behavior you'll add on multiple objects
 const assignee = {
-    getAssignments(){
-        return this.assignments;
-    },
-    setAssignments(deliveries){
-        this.assignments = assignments;
-    }
+  getAssignments() {
+    return this.assignments;
+  },
+  setAssignments(deliveries) {
+    this.assignments = assignments;
+  },
 };
 const aUser = {
-    firstName : 'john',
-    lastName  : 'snow',
-    getName(){
-        return `${this.firstName} ${this.lastName}`;
-    }
+  firstName: "john",
+  lastName: "snow",
```

Comment:
**issue:** please follow the code style in examples 😄 (single quotes only)
**suggestion:**
```suggestion
  firstName: 'john',
  lastName: 'snow',
```

Reply:
Done